backend/src/test/java/com/github/lenaruppert/backend/controller/JobControllerTest.java

```diff
@@ -150,4 +150,27 @@ void whenGetJobsByClientId_thenReturnListOfJobs() throws Exception {
                         ]
                         """));
     }
+
+    @Test
+    @DirtiesContext
+    @WithMockUser(username = "user", password = "password")
+    void whenDeleteJobByIdWithExistingId_thenReturnJobToDelete() throws Exception {
+        clientOne = new Client("1", "nameOfClient", new ArrayList<>());
```

Comment:
Maybe you could include the job's id in the `ArrayList` for a more realistic test